tests/suite/test_app_protect_waf_policies.py

```diff
@@ -384,6 +385,83 @@ def test_ap_waf_policy_logs(
         assert f'request_status="blocked"' in log_contents
         assert f'outcome="REJECTED"' in log_contents
 
+    @pytest.mark.theone
```

Comment:
I'm guessing this was left in by accident 😄

Reply:
indeed:)